scaife_viewer/views.py

```diff
@@ -302,13 +302,16 @@ def search_json(request):
 
 
 def morpheus(request):
-    if "word" not in request.GET:
-        raise Http404()
+    if ("word" not in request.GET) or ("lang" not in request.GET):
+        raise HttpResponseBadRequest()
```

Comment:
Does this work? I believe you need to return not raise.

Reply:
To clarify, I wish this worked :-)

The number of times I've wanted to raise out of some logic and had to rewrite it all...